client/app/account/billing/translations/Messages_fr_FR.xml

```diff
@@ -1176,6 +1176,7 @@
    <translation id="terminate_service_title" qtlid="439957">Formulaire de résiliation</translation>
    <translation id="terminate_service_information_title" qtlid="439970">Résiliation de votre service</translation>
    <translation id="terminate_service_information_text" qtlid="439983">Le service sera résilié dès votre confirmation. Ceci entraînera la suppression automatique, immédiate et irréversible de l’ensemble des données liées à ce service (y compris leurs sauvegardes). Avant de poursuivre et de confirmer votre résiliation, il vous appartient de prendre toutes les mesures nécessaires pour conserver vos données. OVH décline toute responsabilité à ce titre.</translation>
+   <translation id="terminate_service_information_text_non_US">Le service sera résilié à la fin de la période de renouvellement. Une fois que vous confirmez la résiliation, veuillez noter que toutes les données liées à ce service et toutes les sauvegardes <b>seront définitivement supprimées</b>. Il vous appartient de prendre toutes les mesures nécessaires pour conserver vos données avant de donner votre confirmation de supprimer votre service. Une fois le processus initié, vous ne pourrez pas le stopper. OVH décline toute responsabilité de perte de données dans ce cas.</translation>
```

Comment:
What about split your translation in three separate chunk. (In that way avoid using HTML markup and prefer a `<strong>` instead of `<b>` tag as well).

Reply:
My feeling is that's ugly to split the translation. But ok...

Reply:
terminate_service_information_text_non_US_1
terminate_service_information_text_non_US_2
terminate_service_information_text_non_US_3
Is it cleaner?

Reply:
Yes it is! In such way we avoid to deal with HTML tags in translations.

Reply:
You're right! `<strong>` maybe? :)

Reply:
Ok for `strong` :)